src/gemdat/plots/plotly/_shape.py

```diff
@@ -32,4 +34,131 @@ def shape(
     fig : plotly.graph_objects.Figure
         Output figure
     """
-    raise NotImplementedError
+    x_labels = ('X / Å', 'Y / Å', 'Z / Å')
+    y_labels = ('Y / Å', 'Z / Å', 'X / Å')
+
+    fig = make_subplots(
+        rows=2,
+        cols=3,
+        subplot_titles=x_labels,
+        shared_xaxes=True,
+        vertical_spacing=0.1,
+    )
+
+    distances = shape.distances()
+    distances_sq = distances**2
+
+    msd = np.mean(distances_sq)
+    std = np.std(distances_sq)
+    # Latex can not be rendered in 3D plotly plots: https://github.com/plotly/plotly.js/issues/608
+    title = f'{shape.name}: MSD = {msd:.3f} Å^2, std = {std:.3f}'
+
+    mean_dist = np.mean(distances)
+
+    _tmp_dict = defaultdict(list)
+    vector_dict = {}
+    if sites:
+        for site in sites:
+            _tmp_dict[site.label].append(site.coords)
+        for key, values in _tmp_dict.items():
+            vector_dict[key] = np.array(values) - shape.origin
+
+    coords = shape.coords
+
+    for col, (i, j) in enumerate(((0, 1), (1, 2), (2, 0))):
+        x_coords = coords[:, i]
+        y_coords = coords[:, j]
+
+        # Map of the coordinates
+        fig.add_trace(
+            go.Histogram2dContour(
+                x=x_coords,
+                y=y_coords,
+                name=y_labels[col],
+                showscale=False,
+                colorscale='Viridis',
+            ),
+            row=1,
+            col=col + 1,
+        )
```

Comment:
Might be nice to only define the row/col once.

```suggestion
        top_row = {'row': 1, 'col': col + 1}
        bot_row = {'row': 2, 'col': col + 1}

        # Map of the coordinates
        fig.add_trace(
            go.Histogram2dContour(
                x=x_coords,
                y=y_coords,
                name=y_labels[col],
                showscale=False,
                colorscale='Viridis',
            ),
            **top_row
        )
```